service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/NotificationService.java

```diff
@@ -13,15 +13,24 @@
 
 @Slf4j
 @Service
-@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class NotificationService {
     private final NotificationClient notificationClient;
     private final ObjectMapper mapper;
+    private final String environment;
+
+    @Autowired
+    public NotificationService(NotificationClient notificationClient,
+                               ObjectMapper mapper,
+                               @Value("${fpl.env}") String environment) {
```

Comment:
this is not correct, cannot be overridden with env variable configured in chart

Reply:
fixed charts file, thanks for help.

Reply:
if env variable is not present then app wont bootstrap, i believe it is fine, it is better to fail fast rather then produce wrong thing, @akomisarek , do you agree

Reply:
Yes agreed :)